cifar10-outlier-detection/utils/model_training.py

```diff
@@ -0,0 +1,275 @@
+import sys
```

Comment:
These helper files should have one line comments at the top describing what they contain, otherwise it's not totally clear to users why they are part of the example.  Eg something like:

""" Helper functions for training image classification models with AutoGluon and using cross-validation. """

Reply:
I've added it